tasks/lib/taskrunner.js

```diff
@@ -316,7 +316,7 @@ module.exports = function(grunt) {
   Runner.prototype.interrupt = function interrupt() {
     var self = this;
     self._completeQueue();
-    grunt.task.clearQueue();
+    grunt.task.clearQueue({untilMarker: true});
```

Comment:
This works to keep the parent watch task running but the interrupt still ends up killing the karma server 😕  from karma:unit:start. ? Thoughts?